<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>19</y>
      <width>401</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabGalaxy">
     <attribute name="title">
      <string>Galaxy</string>
     </attribute>
     <widget class="QLineEdit" name="leGalaxyName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>321</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Galaxy Parameters</string>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>WindingB</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leWindingB">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leWindingN">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>WindingN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNoArms">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>No Arms</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leArm1">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Arm1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leArm2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Arm2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leArm3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Arm3</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leArm4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>220</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Arm4</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabGalaxy">
     <attribute name="title">
      <string>Components</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Component:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbComponents">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>25</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>291</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Parameters</string>
      </property>
      <widget class="QComboBox" name="cmbComponentType">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>25</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leStrength">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Strength</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Arm</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leArm">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>z0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leZ0">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>r0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leR0">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Delta</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leDelta">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Winding</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leWinding">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Noise Scale</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leScale">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Noise Offset</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNoiseOffset">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>300</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Noise Tilt</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNoiseTilt">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ks</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lePersistence">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>330</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>360</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Is active</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkIsActive">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>360</y>
         <width>87</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>385</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Spectrum</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbSpectrum">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>380</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnNewComponent">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>New Component</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNewComponent_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete component</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Post-processing</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="GLWidget" name="myGLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>571</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="renderButton">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>620</y>
      <width>391</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEhm">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionHvorfor"/>
   </widget>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <addaction name="actionRender"/>
   </widget>
   <addaction name="menuEhm"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRender"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHvorfor">
   <property name="text">
    <string>Hvorfor?</string>
   </property>
  </action>
  <action name="actionRender">
   <property name="text">
    <string>Render</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>source/glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
